itests/testdata/reward_distribution_rolllback_testdata.go

```diff
@@ -40,13 +42,13 @@ func GetRollbackBeforeF19TestData(
 func GetRollbackAfterF19TestData(
 	suite *f.BaseSuite, addresses AddressesForDistribution, height uint64,
 ) RewardDistributionTestData[RewardDistributionExpectedValues] {
-	currentReward := int64(utl.GetCurrentReward(suite, height))
+	currentReward := utl.GetCurrentReward(suite, height)
 	return NewRewardDistributionTestData(
 		addresses,
 		RewardDistributionExpectedValues{
-			MinersSumDiffBalance: currentReward / 3,
-			DaoDiffBalance:       currentReward / 3,
-			XtnDiffBalance:       currentReward / 3,
+			MinersSumDiffBalance: int64(math.Round(float64(currentReward) / 3)),
+			DaoDiffBalance:       int64(math.Round(float64(currentReward) / 3)),
+			XtnDiffBalance:       int64(math.Round(float64(currentReward) / 3)),
```

Comment:
In this case we have to calculate as follows:
```go
p := currentReward / 3
...
                        MinersSumDiffBalance: currentReward - 2*p,
			DaoDiffBalance:       p,
			XtnDiffBalance:       p,
```
actually, the same way as above.

Reply:
Fix